tests/functional/utils/test_os.py

```diff
@@ -70,3 +70,20 @@ def test_get_all_files_in_directory():
         assert len(txt_files) == 3
         assert len(t_txt_files) == 1
         assert len(inner_txt_files) == 1
+
+
+@pytest.mark.parametrize(
+    "path,expected",
+    [
+        ("test.json", ".json"),
+        ("Contract.t.sol", ".t.sol"),
+        ("this.is.a.lot.of.dots", ".is.a.lot.of.dots"),
+        ("directory", ""),
+        (".privateDirectory", ""),
+        ("path/to/.gitkeep", ""),
```

Comment:
also what would something like `.config.json` or something like that do?

probably thinking super nonstandard at this point, as long as we avoid `contracts/.cache` think it's good

Reply:
.cache is ignored.

Reply:
Starting to wish we never did https://github.com/ApeWorX/ape/issues/636

Reply:
Ok made path names like `.cache.json` do `.json`, I think that works... Kinda making up our own path rules at this point.